packages/flame_tiled/analysis_options.yaml

```diff
@@ -2,4 +2,4 @@ include: package:flame_lint/analysis_options.yaml
 
 linter:
   rules:
-    - public_member_api_docs
+    public_member_api_docs: false
```

Comment:
You can just remove this whole section and it will apply the rules from flame_lint (where this rule isn't included).

Reply:
Done.